### DISCUSSION TITLE
Campatability errors

### DISCUSSION THREAD

**Discussion by zvnkk** [JuliaGraphs/MultilayerGraphs.jl#159] [General]
I m encountering compatability errors
>>> add MultilayerGraphs      
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package TensorOperations [6aa20fa7]:
 TensorOperations [6aa20fa7] log:    
 ├─possible versions are: 0.6.1 - 5.1.0 or uninstalled
 ├─restricted by compatibility requirements with MultilayerGraphs [af984530] to versions: 3.2.0 - 3.2.5        
 │ └─MultilayerGraphs [af984530] log:
 │   ├─possible versions are: 0.1.1 - 1.1.4 or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions: 0.1.1 - 1.1.4
 └─restricted by compatibility requirements with CUDA [052768ef] to versions: [0.6.1 - 2.2.0, 4.0.0 - 5.1.0] or uninstalled — no versions left      
   └─CUDA [052768ef] log:
     ├─possible versions are: 0.1.0 - 5.5.2 or uninstalled
     ├─restricted by julia compatibility requirements to versions: [2.3.0, 2.5.0 - 5.5.2] or uninstalled       
     └─restricted by compatibility requirements with GPUCompiler [61eb1bfa] to versions: 5.0.0 - 5.5.2 or uninstalled
       └─GPUCompiler [61eb1bfa] log: 
         ├─possible versions are: 0.1.0 - 1.0.0 or uninstalled
         └─restricted by julia compatibility requirements to versions: 0.23.0 - 1.0.0 or uninstalled

fixing those packages does not work 
could you update pachage to the version where it will automatically build ?

**Comment by angushushu** [comment]
I had a similar issue, it seems the GPU does not support the old version of CUDA.jl, while the other package does not support the new version of CUDA.jl.
